deps/rabbit/Makefile

```diff
@@ -109,6 +109,7 @@ define PROJECT_ENV
 	    {vhost_restart_strategy, continue},
 	    %% {global, prefetch count}
 	    {default_consumer_prefetch, {false, 0}},
+	    {classic_queue_consumer_unsent_message_limit, 200},
```

Comment:
This would also need to be set in `/deps/rabbit/BUILD.bazel`, but see my comment below whether we can get rid of this config option entirely.